api/v1/mapping/src/commonMain/kotlin/Mappings.kt

```diff
@@ -354,6 +354,7 @@ fun OrtRun.mapToApi(jobs: ApiJobs) =
         productId,
         repositoryId,
         revision,
+        resolvedRevision,
```

Comment:
Commit message nits:
- I guess the title intentionally is "compact" and omits "a" before "property", "the" before "resolved revison", and "an" before "ORT run"?
- "store the resolved revision *of the project to analyze*" (or similar) to distinguish from the resolved revision used by the configuration context (which has caused confusion in the past).

Reply:
>     * I guess the title intentionally is "compact" and omits "a" before "property", "the" before "resolved revison", and "an" before "ORT run"?

Yes.

>     * "store the resolved revision _of the project to analyze_" (or similar) to distinguish from the resolved revision used by the configuration context (which has caused confusion in the past).

Done.